fetchai/ledger/genesis/genesis.py

```diff
@@ -103,6 +103,9 @@ def as_json_object(self):
         for (entity, amount, stake_amount) in self._entities_with_wealth:
             token_address = generate_token_address(entity.public_key)
 
+            if not isinstance(amount, int) or not isinstance(stake_amount, int):
+                raise TypeError(f"Initial token amount and stake amount must be integer values! Amount: {amount}, Stake {stake_amount}")
```

Comment:
Format strings aren't supported in python 3.5.
```suggestion
                raise TypeError("Initial token amount and stake amount must be integer values! Amount: {}, Stake {}".format(amount, stake_amount))
```